### ISSUE TITLE
[Bug Report][3.6.9] Accessibility: v-slider doesn't announce custom labels/thumb value

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#20010]
### Environment
**Vuetify Version:** 3.6.9
**Vue Version:** 3.4.27
**Browsers:** Chrome 125.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Use a screen reader (like VoiceOver on Mac) and try to change the value of the slider.

### Expected Behavior
The screen reader should announce the value that the thumb label states (i.e. "Figs', "Lemon", etc.)

### Actual Behavior
 Instead of the displayed value label, the screen reader reads the percent progress the slider thumb has made on the total slider length

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVkDFPwzAQhf/KyUtBajClTJGpxMLUgZ0wmOQACyex7EtaWuW/9xoribvY7313st/dx1kEX8pX5x76DkUuFGHtrCbcFQ2Aqkw/CpZ9Fqyp0EcLkNf6+FKIbSFmQqb8C8yu915/oQ1LMfy2h2xq0Pag/9MioWO8Wci1MwvmhIyfJ7xTckoR48mYT8kkNdtQeuNobMGjaz1Bhd+6swTn+FClSd/dTw7AI3W+WXz8P46QpxjgMYfVm/kJq3VKN0z3WLfNLX5i/I7a39ItU964xRQPsx6iiICNkvM8Ylg3nbXj8XkBjp2ACA==)

### Other comments
I could not find a workaround for this, even when attempting to forcibly set the aria-valuetext to the custom label
